style/base.css

```diff
@@ -21,7 +40,17 @@
 
 .form-class {
   display: flex;
-  background-color: var(--jp-layout-color0);
-  color: var(--jp-ui-font-color0);
+  background-color: var(--jp-layout-color1);
+  color: var(--jp-ui-font-color1);
   font-size: var(--jp-ui-font-size0);
 }
+
+:root {
+  --jp-ui-font-family: 'Arial', 'Dancing Script', 'Dosis', 'Helvetica',
+    'Lobster', 'JetBrains Mono', 'Pacifico', 'Prompt', 'Righteous', 'Sans Serif',
+    'Satisfy', 'Single Day', 'system-ui', 'Times New Roman', 'Urbanist';
+  --jp-content-font-family: 'Arial', 'Dancing Script', 'Helvetica', 'Dosis',
+    'Lobster', 'JetBrains Mono', 'Pacifico', 'Prompt', 'Righteous', 'Sans Serif',
+    'Satisfy', 'Single Day', 'system-ui', 'Times New Roman', 'Urbanist';
+  --jp-code-font-family: 'DejaVu Sans Mono', 'Space Mono';
+}
```

Comment:
You should not override those CSS variables. Otherwise installing your extension is breaking the default themes. Those value should be set only if the user picked a different font family in the theme editor.

```suggestion
```